code/src/index.css

```diff
@@ -11,3 +9,203 @@ code {
+}
+
+.details {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.heart-icon {
+  font-size: 11px;
+  margin: 5px;
+}
+
+button:hover {
+  background-color: #ffd5d5;
+
+}
+
+/*Toughts from API*/
+
+.thought-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  box-sizing: border-box;
+  margin: 20px;
+  padding: 15px;
+  width: 300px;
+  min-height: 112px;
+  background-color: white;
+  border: 1px solid black;
+  box-shadow: 5px 5px black;
+}
+
+
+.thought-details {
+  color: #d0d0d0;
+  font-size: 12px;
+  margin: 0;
+}
+
+.thought-text {
+  font-size: 13px;
+  font-weight: bold;
+  margin: 0;
+  margin-bottom: 10px;
+  overflow-wrap: break-word;
+}
+
+.like-container {
+  display: inline-flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.like-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  margin-right: 10px;
+  width: 35px;
+  height: 35px;
+  border: none;
+  border-radius: 50%;
+  background-color: #FFADAD;
+  font-size: 11px;
+  cursor: pointer;
+}
+
+.noLikes {
+  background-color: #e9e8e8;
+}
+
+/*New Thought*/
+
+.new-thought {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  box-sizing: border-box;
+  margin: 20px;
+  padding: 15px;
+  width: 300px;
+  height: auto;
+  background-color: #F2F0F0;
+  border: 1px solid black;
+  box-shadow: 5px 5px black;  
+}
+
+.new-thought p {
+  font-size: 12px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.submit-thought {
+  width: 200px;
+  padding: 12px;
+  background-color: #FFADAD;
+  border: none;
+  border-radius: 30px;
+}
+
+textarea {
+  width: 100%;
+  margin: 10px 0 5px 0;
+  box-sizing: border-box;
+  resize: vertical;
+}
+
+.warning {
+  color: red;
+}
+
+.character-count {
+  text-align: right;
+}
+
+.new-thought .details {
+  margin-bottom: 10px;
+}
+
+ /* LOADING */ 
+
+.loading-container {
+  display: inline-flex;
+  justify-content: center;
+  width: 100px;
+  height: auto;
+}
+
+.loading-circle {
+  width: 10px;
+  height: 10px;
+  background-color: #FFADAD;
+  border-radius: 50%;
+  margin: 5px;
+  animation: loading 0.6s ease infinite;
+}
+
+.loading-container :nth-child(1) {
+  animation-delay: 0s;
+}
+
+.loading-container :nth-child(2) {
+  animation-delay: 0.2s;
+}
+
+.loading-container :nth-child(3) {
+  animation-delay: 0.4s;
+}
+
+@keyframes loading {
+  0% {background-color: #ffd5d5;}
+  50% {background-color: #FFADAD;}
+  100% {background-color: #ffd5d5;}
+}
+
+@media screen and (min-width: 550px) {
+  .new-thought {
+    width: 500px;
+    min-height: 150px;
+  }
+
+  .thought-container {
+    width: 500px;
+    min-height: 150px;
+  }
+
+  .thought-details {
+    font-size: 15px;
+  }
+
+  .thought-text {
+    font-size: 16px;
+  }
+
+  .new-thought p {
+    font-size: 15px;
+  }
+
+  .submit-thought {
+    width: 240px;
+  }
+
+  textarea {
+    min-height: 75px;
+    font-size: 15px;
+  }
```

Comment:
And it looks great in both mobile, tablet and desktop!

Reply:
Forgot to write that I checked the list of requirements and stretch-goals, and you got them all, except the one showing a counter how many posts you have liked, it would have been fun to see how you could have solved that :)